doc/faq.md

```diff
@@ -176,6 +176,29 @@ re-install Convos:
 
 Please [contact us](/doc/#get-in-touch) if the above instructions do not work.
 
+## Why convos does not restart after an upgrade of convos code?
+
+Some changes in the convos code do not require any special operation (for instance if you modify only assets and/or themes). 
+
+If you update the convos code and get stuck to the loading page after a refresh like this :
```

Comment:
I'm not a fan of space in front of colon. (This occurs multiple times in this PR)

Reply:
I think I fixed this